compose.go

```diff
@@ -55,15 +56,43 @@ type LocalDockerCompose struct {
 	WaitStrategyMap      map[waitService]wait.Strategy
 }
 
+type (
+	LocalDockerComposeOptions struct {
+		Logger Logging
+	}
+	LocalDockerComposeOption interface {
+		Apply(opts *LocalDockerComposeOptions)
+	}
+	LocalDockerComposeOptionsFunc func(opts *LocalDockerComposeOptions)
+)
+
+func (f LocalDockerComposeOptionsFunc) Apply(opts *LocalDockerComposeOptions) {
+	f(opts)
+}
+
+func LocalComposeWithLogger(logger Logging) LocalDockerComposeOption {
+	return LocalDockerComposeOptionsFunc(func(opts *LocalDockerComposeOptions) {
+		opts.Logger = logger
+	})
+}
+
 // NewLocalDockerCompose returns an instance of the local Docker Compose, using an
 // array of Docker Compose file paths and an identifier for the Compose execution.
 //
 // It will iterate through the array adding '-f compose-file-path' flags to the local
 // Docker Compose execution. The identifier represents the name of the execution,
 // which will define the name of the underlying Docker network and the name of the
 // running Compose services.
-func NewLocalDockerCompose(filePaths []string, identifier string) *LocalDockerCompose {
-	dc := &LocalDockerCompose{}
+func NewLocalDockerCompose(filePaths []string, identifier string, opts ...LocalDockerComposeOption) *LocalDockerCompose {
```

Comment:
This change would break the API of those using this method. Have you though maybe creating a fluent builder pattern, something like:

```golang
func (dc *LocalDockerCompose) WithComposeOptions(opts ...LocalDockerComposeOption) *LocalDockerCompose {
     ...
    return dc
}

This way, consumers won't be affected and could be able to customise the compose instance if needed 🤔

Reply:
:smile: Honestly I'm not such a huge fan of fluent builders in Go due to the constraints where I **must** put the `.` :smile:  but if you prefer it that way I can try to refactor it :blush: 

As you already noticed I only updated the tests to already use the logger option both for better output and demonstration :blush: 

I think this should not break the API, does it? :thinking:

Reply:
Nope, I updated my comment in UPDATE 2 :) So it's fine for me in terms of that API. Will take a deeper look with the description you provided, thanks for updating it

Reply:
ah sorry, didn't check when you updated the first comment :sweat_smile: